pkg/events/dispatcher.go

```diff
@@ -136,6 +155,7 @@ func NewHandler(ctx context.Context, repo *database.Repo, azureClient *auth.Azur
 		teamClient:  teamClient,
 		userClient:  user.NewClient(repo, gcpProject, gcpRegion, gcpZone, dryRun),
 		chartClient: chartClient,
+		helmClient:  helm.NewClient(dryRun, repo),
 	}, nil
 }
 
```

Comment:
Ikke en kommentar til denne linjen, men skulle vi heller krevd at `Run()` tok inn `contecxt` i stedet for at vi setter den på `EventHandler`, for er vel ingen som trenger `e.context`?

Reply:
Tja, i `processWork` så trenger vi to contexter. En med timeout for eventen og en for å sette status for eventen etterpå. Vi kunne sikkert brukt contexten med timeout til å sette pending (ved retry) og completed (ved suksess), men ved failed status kan det potensielt være fordi helm upgrade f.eks. har timet ut og da er ikke denne contexten gyldig lenger og vi vil ikke få satt event statusen i databasen.